src/benchmark.cc

```diff
@@ -575,6 +621,13 @@ void BenchmarkImp::Repetitions(int n) {
   repetitions_ = n;
 }
 
+void BenchmarkImp::ReportAggregatesOnly(bool value) {
+  if (report_options_ == RO_Unspecified)
+      report_options_ = RO_Default;
+  if (value) report_options_ &= ~RO_ReportAggregatesOnly;
```

Comment:
this is very short and neat, but it requires a lot of work above. can you just be more explicit here about the state changes and lose all the operator overload cleverness please?